types/node-telegram-bot-api/index.d.ts

```diff
@@ -5,6 +5,7 @@
 //                 Giorgio Garasto <https://github.com/Dabolus>
```

Comment:
Does the package version need to be updated (line 1)?

Reply:
No, the package hasn't been updated.
Although, I am confused since the original package is in version 0.30 and this type definition says it is for v0.31

Reply:
[NPM](https://www.npmjs.com/package/node-telegram-bot-api)
[Github](https://github.com/yagop/node-telegram-bot-api/blob/master/package.json)

Reply:
Weird. https://github.com/yagop/node-telegram-bot-api/pull/752.

So, it looks like the changes you’ve made here exist on GitHub, but not yet on npm, but maybe will be published soon. But that means there _is_ a version mismatch here—I think you need to bump the version to 0.40, and maybe we should wait until the actual library gets published.